controllers/users/logout.js

```diff
@@ -0,0 +1,30 @@
+const { Unauthorized } = require('http-errors');
+const User = require('../../models/users');
+
+const logout = async (req, res, next) => {
+  try {
+    console.log("HOLA")
+    // Obtener el ID del usuario desde el token
+    const userId = req.user.userId;
```

Comment:
Aca accederiamos a user id usando `_id`
```suggestion
    const userId = req.user._id;
```